manifests/init.pp

```diff
@@ -29,6 +31,8 @@
   $slaveof            = undef,
   $version            = 'installed',
   $packages           = ['redis']
+  $redis_conf         = 'redis.conf',
```

Comment:
Can you make these variables represent full paths to the config file?  Its odd to make just the filename a param.